assets/src/components/activities/likert/LikertAuthoring.tsx

```diff
@@ -0,0 +1,109 @@
+  const selectedPartId = model.authoring.parts[0].id;
+  const selectedItem = model.items.find((i) => i.id == selectedPartId) || model.items[0];
+
+  return (
+    <React.Fragment>
+      <TabbedNavigation.Tabs>
+        <TabbedNavigation.Tab label="Question">
+          <Stem />
+          <div>
+            <br />
+            <p>Choices:</p>
+            <ChoicesAuthoring
+              icon={<Radio.Unchecked />}
+              choices={model.choices}
+              setAll={(choices: ActivityTypes.Choice[]) => dispatch(Choices.setAll(choices))}
+              onEdit={(id, content) => dispatch(Choices.setContent(id, content))}
+              addOne={() => dispatch(LikertActions.addChoice())}
+              onRemove={(id) => dispatch(LikertActions.removeChoice(id))}
+            />
+          </div>
+          <p>Questions:</p>
+          <ChoicesAuthoring
+            choices={model.items}
+            setAll={(choices: ActivityTypes.Choice[]) => dispatch(Items.setAll(choices))}
+            onEdit={(id, content) => dispatch(Items.setContent(id, content))}
+            addOne={() => dispatch(LikertActions.addItem())}
+            onRemove={(id) => dispatch(LikertActions.removeItem(id))}
+          />
+        </TabbedNavigation.Tab>
+        <TabbedNavigation.Tab label="Answer Key">
+          <StemDelivery stem={selectedItem} context={defaultWriterContext()} />
+
+          <ChoicesDelivery
+            unselectedIcon={<Radio.Unchecked />}
+            selectedIcon={<Radio.Checked />}
+            choices={model.choices}
+            selected={[getCorrectChoice(model, selectedPartId).id]}
+            onSelect={(id) => dispatch(MCActions.toggleChoiceCorrectness(id, selectedPartId))}
+            isEvaluated={false}
+            context={defaultWriterContext()}
+          />
+          <SimpleFeedback partId={selectedPartId} />
+          <TargetedFeedback
+            toggleChoice={(choiceId, mapping) => {
+              dispatch(MCActions.editTargetedFeedbackChoice(mapping.response.id, choiceId));
+            }}
+            addTargetedResponse={() => dispatch(MCActions.addTargetedFeedback())}
+            unselectedIcon={<Radio.Unchecked />}
+            selectedIcon={<Radio.Checked />}
+          />
+        </TabbedNavigation.Tab>
+
+        <TabbedNavigation.Tab label="Hints">
+          <Hints partId={selectedPartId} />
+        </TabbedNavigation.Tab>
+      </TabbedNavigation.Tabs>
+    </React.Fragment>
+  );
+};
+
+const store = configureStore();
+
+export class LikertAuthoring extends AuthoringElement<LikertModelSchema> {
+  render(mountPoint: HTMLDivElement, props: AuthoringElementProps<LikertModelSchema>) {
+    ReactDOM.render(
+      <Provider store={store}>
+        <AuthoringElementProvider {...props}>
+          <Likert {...props} />
+        </AuthoringElementProvider>
+      </Provider>,
+      mountPoint,
+    );
+  }
+}
+// eslint-disable-next-line
```

Comment:
We should be sparing `// eslint-disable-next-line` and imports should be at the top of the file.

I'm guessing this was copied from an existing reference activity so I'm guessing there was a good reason for it and is fine for now, we can take a task later to clean these up if necessary